drafttopic/feature_lists/wikidata.py

```diff
@@ -0,0 +1,24 @@
+from revscoring.datasources.meta import vectorizers, mappers
+from revscoring.features import modifiers
+from revscoring.features import wikibase
+from revscoring.features.meta import aggregators
+
+
+wikidata_kvs = vectorizers.word2vec.load_gensim_kv(
+    filename="wikidata-20200501-learned_vectors.50_cell.10k.kv", mmap="r")
+
+
+def vectorize_words(words):
+    return vectorizers.word2vec.vectorize_words(wikidata_kvs, words)
+
+
+revision_text_vectors = vectorizers.word2vec(
+    wikibase.revision.datasources.claims,
```

Comment:
@halfak I noticed that this dependency does almost what we want, but it gets pid and datavalue instead of pid and datavalue.id. So I am assuming we need to duplicate this in revscoring with slight modification and make necessary changes here?

Reply:
Wooops!  I thought I'd responded to this earlier.  Might have forgotten to hit send.   

I think we can use `wikibase.revision.datasources.claims`.  We'll need to define an intermediate datasource that will let us get a stream of Pids and Qids.  I imagine something like this:


```
QID_RE = re.compile("Q[0-9]+")

def process_claims_to_words(claims):
  words = []
  for pid, value in claims:
    words.append(pid)
    if QID_RE.match(value):
      words.append(value)
  return words

claim_words = Datasource(
  "wikidata.revision.claim_words", process_claims_to_words,
  depends_on=[wikibase.revision.datasources.claims])
```

Reply:
Thanks @halfak!